### ISSUE TITLE
Improve changelog tooling

### ISSUE THREAD

**Issue by dbkr** [vector-im/element-web#6613]
* `github-changelog-generator` is unfeasibly slow
 * It doesn't deal with commits cherry-picked from develop to release branches
 * Almost all the changes end up in the changelog for the first RC
 * Since 99% of our changes are in react-sdk, riot-web's changelog is pretty uninformative

**Comment by lampholder** [comment]
Does this slow down our release process?

**Comment by shirishag75** [comment]
the downside of not having is, the only changelog we get is - 

```
/usr/share/doc/riot-web$ zcat changelog.gz 
riot-web (0.14.2) whatever; urgency=medium

  * Package created with FPM.

 -- support@riot.im  Mon, 30 Apr 2018 13:54:49 +0100

```

**Comment by dbkr** [comment]
@shirishag75 from my POV, the bug you created (https://github.com/vector-im/riot-web/issues/4879) is for tracking getting changelogs into the debian package, which is a separate issue.

**Comment by jryans** [comment]
I recorded [some thoughts on a possible way forward](https://docs.google.com/document/d/1_gDYLAXuly561ylyBzetP1sj08eeMNOVes927QUNXD0/edit) here, but unfortunately I haven't found time to actually make it happen yet. 😅 Perhaps the notes will be useful for someone else.

**Comment by shirishag75** [comment]
Good. Is vector be part of GSOC 2021. If yes, then perhaps it could be one of the tasks for a would-be intern.

**Comment by SimonBrandner** [comment]
> Good. Is vector gonna be part of GSOC 2021. If yes, then perhaps it could be one of the tasks for a would-be intern.

GSOC 2021 is already in progress, so this would have to be next year :(

**Comment by dbkr** [comment]
Now fixed! (Leaving #4879 open because we still need to hook the changelog up to be in the debian package changelog)
